packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1867,6 +1918,112 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   }
 
+  /**
+   * Set up the listeners for captions
+   * @returns {undefined}
+   * @private
+   * @memberof Meeting
+   */
+  private setUpVoiceaListener() {
+    // @ts-ignore
```

Comment:
This is because, `this.webex` won't be typed and is not the right way to use inside Typescript. However we can only do it this way for internal plugins as they use Ampstate and are not modules.